# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1296211632 18000
#      Fri Jan 28 05:47:12 2011 -0500
# Branch release-3-4-x
# Node ID 9290d2086b5f40c56cdb1fac26ad7e8ce744ce11
# Parent  587c99975455cec920ad71b2cb5e08fc71ad7525
pr-output.cc (Foutput_max_field_width): fix cut and paste error

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-28  John W. Eaton  <jwe@octave.org>
+
+	* pr-output.cc (Foutput_max_field_width): Set internal variable
+	output_max_field_width, not output_precision.
+
 2011-01-28  John W. Eaton  <jwe@octave.org>
 
 	* pr-output.cc (Fformat, Ffixed_point_format, Foutput_precision,
 	Foutput_max_field_width, Fsplit_long_rows, Fprint_empty_dimensions):
 	Update @seealso lists.
 
 2011-01-27  Rik  <octave@nomad.inbox5.com>
 
diff --git a/src/pr-output.cc b/src/pr-output.cc
--- a/src/pr-output.cc
+++ b/src/pr-output.cc
@@ -4009,17 +4009,17 @@ DEFUN (output_max_field_width, args, nar
   "-*- texinfo -*-\n\
 @deftypefn  {Built-in Function} {@var{val} =} output_max_field_width ()\n\
 @deftypefnx {Built-in Function} {@var{old_val} =} output_max_field_width (@var{new_val})\n\
 Query or set the internal variable that specifies the maximum width\n\
 of a numeric output field.\n\
 @seealso{format, fixed_point_format, output_precision}\n\
 @end deftypefn")
 {
-  return SET_INTERNAL_VARIABLE_WITH_LIMITS (output_precision, 0, INT_MAX);
+  return SET_INTERNAL_VARIABLE_WITH_LIMITS (output_max_field_width, 0, INT_MAX);
 }
 
 DEFUN (output_precision, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn  {Built-in Function} {@var{val} =} output_precision ()\n\
 @deftypefnx {Built-in Function} {@var{old_val} =} output_precision (@var{new_val})\n\
 Query or set the internal variable that specifies the minimum number of\n\
 significant figures to display for numeric output.\n\
